shared/fs/row/load-files-when-needed.js

```diff
@@ -9,52 +9,67 @@ type OwnProps = {|
   destinationPickerIndex?: number,
 |}
 
+const mapStateToProps = state => ({
+  syncingFoldersProgress: state.fs.syncingFoldersProgress,
+})
+
 const mapDispatchToProps = (dispatch, {path, destinationPickerIndex}) => ({
   loadFavorites: () => dispatch(FsGen.createFavoritesLoad()),
   loadFolderList: () =>
     dispatch(
       FsGen.createFolderListLoad({
         path,
-        refreshTag: typeof destinationPickerIndex === 'number' ? 'destination-picker' : 'main',
       })
     ),
 })
 
-const mergeProps = (stateProps, {loadFolderList, loadFavorites}, {path}) => ({
-  loadFavorites,
-  loadFolderList,
+const mergeProps = (s, d, {path}) => ({
+  ...s,
+  ...d,
   path,
 })
 
 type Props = {|
-  loadFolderList: () => void,
+  loadFolderList: (setRefreshTag: boolean) => void,
```

Comment:
Instead of being a `boolean`, could this be a `'set-refresh-tag' | 'no-set-refresh-tag'` or something, so that the code calling `_load(true)` and such is more legible?

Reply:
I felt like it was fine as this is in the same file. But I guess it wouldn't hurt to define some constants inside the file.

Reply:
/shrug I don't have strong feelings.